calico/felix/endpoint.py

```diff
@@ -585,7 +604,7 @@ def _on_worker_died(self, watch_greenlet):
 
 class LocalEndpoint(RefCountedActor):
 
-    def __init__(self, config, combined_id, ip_type, iptables_updater,
+    def __init__(self, config, combined_id, ip_type, iptables_updater, untracked_updater,
```

Comment:
Fair point.  But sounds like on balance you're saying leave this as is, so I'll do that.